### DISCUSSION TITLE
Is prisma spec language https://github.com/prisma/specs/tree/master/schema are deprecated?

### DISCUSSION THREAD

**Discussion by mandaputtra** [prisma/prisma#5283] [Q&A] [answered]
I came across this spec https://github.com/prisma/specs/tree/master/schema and its now archived, is this still used? Seems like the syntax are much different from the [ofc docs](https://www.prisma.io/docs/concepts/components/prisma-schema)?  But still mentioned in the [ofc docs](https://www.prisma.io/docs/concepts/components/prisma-schema#syntax)

**Comment by nikolasburk** [comment, marked as answer]
Hey @mandaputtra 👋  the entire `specs` repo has indeed been deprecated, so the spec for the PSL is also not in use any more. It might still serve to provide some background on some decisions we've made, but it doesn't count as "official" information any more. I also just removed the link from the docs 🙌 

Let me know if you have any further questions, always happy to help :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
